src/main/java/controllers/ConvertModel.java

```diff
@@ -0,0 +1,17 @@
+package controllers;
+
+import dto.UserResponseDto;
+import model.User;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConvertModel {
```

Comment:
```suggestion
public class UserMapper {
```
class name  should be noun